HELPS/ClientApp/src/util.ts

```diff
@@ -0,0 +1,30 @@
+
+export function isEmpty<T>(object: string | T[]): boolean {
+    return object.length === 0;
+}
+
+export const NOOP = () => ({});
+
+export const fetchRequest = <T>(
```

Comment:
This is handy, we might replace the token parameter with some global fetch within the function to add some consistency